pkgs/applications/video/simplescreenrecorder/default.nix

```diff
@@ -24,8 +24,8 @@ stdenv.mkDerivation rec {
     done
   '';
 
-  nativeBuildInputs = [ pkgconfig ];
-  buildInputs = [  
+  nativeBuildInputs = [ pkgconfig cmake ];
+  buildInputs = [
     alsaLib ffmpeg libjack2 libX11 libXext libXfixes mesa
     libpulseaudio qt4
   ];
```

Comment:
`enableParallelBuilding = true;` no longer needed, cmake builds in parallel by default.